scripts/installers/make-macpkg

```diff
@@ -20,13 +20,17 @@ ROOT = os.path.dirname(
 PKG_DIR = os.path.join(ROOT, 'macpkg')
 SCRIPTS_DIR = os.path.join(PKG_DIR, 'scripts')
 RESOURCES_DIR = os.path.join(PKG_DIR, 'resources')
-DISTRIBUTION_PATH = os.path.join(PKG_DIR, 'distribution.xml')
 
 TEMP_PKG_NAME = 'aws-cli.pkg'
 PKG_NAME = 'AWS-CLI-Installer.pkg'
 EXE_ZIP_NAME = 'awscli-exe.zip'
 EXE_NAME = 'aws'
 
+def select_distribution_path():
+    base = os.path.join(PKG_DIR, 'distribution.xml')
+    uni = os.path.join(PKG_DIR, 'distribution.universal.xml')
```

Comment:
What's the rationale for maintaining 2 separate distribution files? What happens if we update `distribution.xml` with the host architectures and then produce an x86-only installer?